sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkExecuteStatementOperation.scala

```diff
@@ -142,7 +144,9 @@ private[hive] class SparkExecuteStatementOperation(
           } catch {
             case NonFatal(e) =>
               setOperationException(new HiveSQLException(e))
-              logError(s"Error cancelling the query after timeout: $timeout seconds")
+              val timeout_ms = timeout * MILLIS_PER_SECOND
```

Comment:
qq, just for my personal understanding: why do we convert second to millisecond here?

Reply:
https://github.com/apache/spark/pull/45903/